isis/tests/Fixtures.h

```diff
@@ -174,6 +174,12 @@ namespace Isis {
     protected:
       void setInstrument(QString ikid, QString instrumentId, QString spacecraftName); 
   };
+
+
+  class ApolloCube : public LargeCube { 
+    protected: 
+      void SetUp() override; 
+  };
```

Comment:
Do we want to use the `SetUp` function here or `setInstrument`? For consistency we have used the setInstrument function for both Mro and NewHorizons so we may want to continue that here.

Reply:
The set instrument stuff it partly because instruments that used it similar instrument groups so simply changing things like the IKID can allow use to make cubes of different instruments from the same mission. As far as I know the same isn't true for Apollo?